src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/payment/FeesAndStatusCheckPreparerTest.java

```diff
@@ -1,344 +1,345 @@
-    }
-
-    private static Stream<Arguments> paymentOptionNotAvailable() {
-        return Stream.of(
-            Arguments.of(Optional.of(PaymentStatus.PAID), Optional.empty()),
-            Arguments.of(Optional.of(PaymentStatus.PAID), Optional.of(RemissionDecision.APPROVED)),
-            Arguments.of(Optional.of(PaymentStatus.PAID), Optional.of(RemissionDecision.PARTIALLY_APPROVED)),
-            Arguments.of(Optional.of(PaymentStatus.PAYMENT_PENDING), Optional.of(RemissionDecision.APPROVED)),
-            Arguments.of(Optional.of(PaymentStatus.PAYMENT_PENDING), Optional.of(RemissionDecision.PARTIALLY_APPROVED))
-        );
-    }
-
-    @ParameterizedTest
-    @MethodSource("paymentOptionNotAvailable")
-    void should_error_when_payment_option_should_be_unavailable(
-        Optional<PaymentStatus> paymentStatus,
-        Optional<RemissionDecision> remissionDecision) {
-
-        when(callback.getEvent()).thenReturn(Event.PAYMENT_APPEAL);
-        when(callback.getCaseDetails()).thenReturn(caseDetails);
-        when(callback.getCaseDetails().getCaseData()).thenReturn(asylumCase);
-        when(asylumCase.read(APPEAL_TYPE, AppealType.class)).thenReturn(Optional.of(AppealType.EA));
-        when(asylumCase.read(PAYMENT_STATUS, PaymentStatus.class)).thenReturn(paymentStatus);
-        when(asylumCase.read(REMISSION_DECISION, RemissionDecision.class)).thenReturn(remissionDecision);
-
-        PreSubmitCallbackResponse<AsylumCase> callbackResponse =
-            feesAndStatusCheckPreparer.handle(PreSubmitCallbackStage.ABOUT_TO_START, callback);
-
-        assertNotNull(callbackResponse);
-        assertThat(callbackResponse.getErrors()).isNotEmpty();
-        assertThat(callbackResponse.getErrors())
-            .contains("The Make a payment option is not available.");
-    }
-
-    @ParameterizedTest
-    @ValueSource(strings = {"refusalOfEu", "refusalOfHumanRights", "protection", "euSettlementScheme"})
-    void should_error_when_remission_approved_or_partially_approved(String type) {
-
-        when(callback.getEvent()).thenReturn(Event.PAYMENT_APPEAL);
-        when(callback.getCaseDetails()).thenReturn(caseDetails);
-        when(callback.getCaseDetails().getCaseData()).thenReturn(asylumCase);
-        when(asylumCase.read(APPEAL_TYPE, AppealType.class)).thenReturn(AppealType.from(type));
-        when(asylumCase.read(PAYMENT_STATUS, PaymentStatus.class)).thenReturn(Optional.of(PaymentStatus.PAID));
-
-        PreSubmitCallbackResponse<AsylumCase> callbackResponse =
-            feesAndStatusCheckPreparer.handle(PreSubmitCallbackStage.ABOUT_TO_START, callback);
-
-        assertNotNull(callbackResponse);
-        assertThat(callbackResponse.getErrors()).isNotEmpty();
-        assertThat(callbackResponse.getErrors())
-            .contains("The Make a payment option is not available.");
-    }
-
-}
+package uk.gov.hmcts.reform.iacaseapi.domain.handlers.presubmit.payment;

```

Comment:
Can you fix the formatting here too? It's hard to review when the whole file has diff

Reply:
done